docker/test/integration/features/http.feature

```diff
@@ -8,7 +8,6 @@ Feature: Sending data using InvokeHTTP to a receiver using ListenHTTP
 
   Scenario: A MiNiFi instance transfers data to another MiNiFi instance with message body
     Given a GetFile processor with the "Input Directory" property set to "/tmp/input"
-    And the "Keep Source File" property of the GetFile processor is set to "true"
```

Comment:
Are these removed because the input files are not used anymore and just to keep things simple? Or was there an actual problem due to processing the same file multiple time? Or what is the reason for this change?

Reply:
There was no problem but it seemed unnecessary so the change was made to just keep it simple :)